wallet/ios/Runner/AppDelegate.swift

```diff
@@ -9,9 +9,12 @@ import flutter_downloader
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
   ) -> Bool {
-    // Uncomment while using ios simulator 
-//     let providerFactory = AppCheckDebugProviderFactory()
-//     AppCheck.setAppCheckProviderFactory(providerFactory)
+    // Uncomment while using ios simulator
+
+     let providerFactory = AppCheckDebugProviderFactory()
+     AppCheck.setAppCheckProviderFactory(providerFactory)
```

Comment:
add an environment variable and use an "if" statement here instead of commenting.